src/method.c

```diff
@@ -33,7 +33,7 @@ jl_value_t *jl_resolve_globals(jl_value_t *expr, jl_module_t *module, jl_svec_t
         }
         else {
             if (e->head == call_sym && jl_expr_nargs(e) == 3 &&
-                    jl_is_quotenode(jl_exprarg(e, 2)) && module != NULL) {
+                jl_is_quotenode(jl_exprarg(e, 2)) && module != NULL) {
```

Comment:
it's split pretty evenly depending on who touched it most recently. @vtjnash has been going in the extra indentation direction pretty consistently with code he touches